__tests__/app.test.js

```diff
@@ -99,9 +99,44 @@ describe("app", () => {
     })
   })
  });
+ describe("GET /api/articles", () => {
+  test("200: responds with a status of 200 when article objects are found", () => {
+    return request(app).get(`/api/articles`).expect(200);
+  });
+  test("200: responds with correct comment_count on related actricle_id", () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
+      const articleIdToTest = 34;
+      const article = response.body.articles.filter(article => article.article_id === articleIdToTest);
+      const expectedCommentCount = 11;
+    });
+  });
+  test("200: respond with an article object with the following properties when article objects are found", () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
+      const {articles} = response.body;
+      articles.forEach((article) => {
+      expect(article).toBeInstanceOf(Object);
+      expect(article).toHaveProperty("author", expect.any(String));
+      expect(article).toHaveProperty("title", expect.any(String));
+      expect(article).toHaveProperty("article_id", expect.any(Number));
+      expect(article).toHaveProperty("topic", expect.any(String));
+      expect(article).toHaveProperty("created_at", expect.any(String));
+      expect(article).toHaveProperty("votes", expect.any(Number));
+      expect(article).toHaveProperty("article_img_url", expect.any(String));
+      expect(article).toHaveProperty("comment_count", expect.any(String));
+      expect(article).not.toHaveProperty("body");
+    });
+  });
+ });
```

Comment:
Good start. We also need to test if these articles are returned, by default, in descending order by created_at